Sources/SwiftDocC/Utility/FeatureFlags.swift

```diff
@@ -28,7 +28,7 @@ public struct FeatureFlags: Codable {
     public var isExperimentalMentionedInEnabled = false
     
     /// Whether or not experimental support validating parameters and return value documentation is enabled.
-    public var isExperimentalParametersAndReturnsValidationEnabled = false
+    public var isExperimentalParametersAndReturnsValidationEnabled = true
```

Comment:
Should we rename this to `isParametersAndReturnsValidationEnabled` ? Is this no longer experimental?

Reply:
Yes. Since this is both enabled-by-default and documented it should be considered no longer experimental.